lightning/src/routing/scoring.rs

```diff
@@ -391,12 +402,14 @@ pub struct ProbabilisticScoringParameters {
 	/// [`liquidity_offset_half_life`]: Self::liquidity_offset_half_life
 	pub liquidity_penalty_multiplier_msat: u64,
 
-	/// The time required to elapse before any knowledge learned about channel liquidity balances is
-	/// cut in half.
+	/// We track upper and lower bounds on the available liquidity in a channel in order to
+	/// calculate success probability given recent successes and failures. In order to ensure we do
+	/// not get stuck avoiding channels which only failed once, we decay these bounds over time.
```

Comment:
Saying "only failed once" gives a limited picture given the calculation is dependent on the amount. Seems like the re-write here is only focusing on a specific example rather than being more general. Not sure how necessary this is if the struct-level docs are clearer, to be honest.